src/main/kotlin/ai/digital/integration/server/deploy/tasks/kube/scanning/KubeAwsScannerFinalizerTask.kt

```diff
@@ -0,0 +1,29 @@
+package ai.digital.integration.server.deploy.tasks.kube.scanning
+
+import ai.digital.integration.server.common.util.KubeCtlUtil
+import ai.digital.integration.server.common.util.ProcessUtil
+import ai.digital.integration.server.common.util.KubeScanningUtil
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+import java.io.File
+
+open class KubeAwsScannerFinalizerTask : DefaultTask() {
+
+    companion object {
+        const val NAME = "KubeAwsScannerFinalizer"
+    }
+
+    @TaskAction
+    fun launch() {
+        KubeCtlUtil.delete(project, File("${KubeScanningUtil.getKubeBenchDir(project)}/job-eks.yaml"))
```

Comment:
It's hard to read such code.
It's better to move each command to a separate method and name it in a human readable way.

Reply:
Same applies for the code below. 
It takes unnecessary effort, to understand what each command does.